brian2/devices/device.py

```diff
@@ -412,10 +414,10 @@ def spike_queue(self, source_start, source_end):
         # Use the C++ version of the SpikeQueue when available
         try:
             from brian2.synapses.cythonspikequeue import SpikeQueue
-            logger.info('Using the C++ SpikeQueue', once=True)
+            logger.diagnostic('Using the C++ SpikeQueue', once=True)
         except ImportError:
             from brian2.synapses.spikequeue import SpikeQueue
-            logger.info('Using the Python SpikeQueue', once=True)
+            logger.diagnostic('Using the Python SpikeQueue', once=True)
```

Comment:
I agree with making the C++ SpikeQueue be at diagnostic level, but maybe if they are using the Python one that should be at debug level because it will be problematic for performance. I'm also wondering if we could have a common bit of text for performance information, either in the message or in the log item name)? Like that you could quickly grep/search the log for performance related information.